mopidy/core/actor.py

```diff
@@ -136,6 +138,106 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def on_start(self):
+        logger.debug("core on_start")
+        try:
+            coverage = []
+            if self._config and 'restore_state' in self._config['core']:
+                amount = self._config['core']['restore_state']
```

Comment:
Nitpicking, but amount is more a numeric thing in my head, value fits better IMO.